package.json

```diff
@@ -240,7 +646,8 @@
     "react-dom": "18.3.1",
     "semver": "7.5.2",
     "dset": "^3.1.4",
-    "cross-spawn": "^7.0.5"
+    "cross-spawn": "^7.0.5",
+    "yaml": "^2.6.1"
```

Comment:
why do we need to add it in the "resolutions"? There is a conflict with another library?

Reply:
https://github.com/KaotoIO/kaoto/blob/main/packages/ui/package.json#L76